content/en/faqitems/security-1-immutability-files-change-on-host.markdown

```diff
@@ -0,0 +1,14 @@
++++
+question = "Does Bottlerocket’s immutability mean that no files will ever change on the host?"
```

Comment:
Can you rework the question to better fit answer? (The answer is good.. question maybe could use work).